tensorbay/client/segment.py

```diff
@@ -328,6 +329,43 @@ def _upload_label(self, data: Union[AuthData, Data]) -> None:
 
         self._client.open_api_do("PUT", "labels", self._dataset_id, json=post_data)
 
+    def _upload_multi_label(self, data: Iterable[_D]) -> None:
+        post_data: Dict[str, Any] = {
+            "segmentName": self.name,
+            "objects": [],
+        }
+        for single_data in data:
+            label = single_data.label.dumps()
+            if not label:
+                continue
+
+            remote_path = (
+                single_data.path
+                if isinstance(single_data, RemoteData)
+                else single_data.target_remote_path
+            )
+            post_data["objects"].append({"remotePath": remote_path, "label": label})
```

Comment:
calling `__getitem__` in a for loop which is slow, a better way is use a local variable `objects = []`, append labels into `objects` and set `post_data["objects"] = object` outside the for-loop